setup.cfg

```diff
@@ -1,5 +1,3 @@
-[pycodestyle]
-[pep8]
```

Comment:
Can you check that nobody actually uses these tools in their editors? I don't know about PyCharm users etc. which sections are supported and which aren't.

Reply:
I believe `pycodestyle` needs to be installed for it to be picked up by IDEs. Anyway, we don't enforce it at the CI level

Reply:
Ah wait maybe I'll then need to edit some of the `ignore` rules that are not relevant anymore.

Reply:
> Anyway, we don't enforce it at the CI level

We don't care which tools people use locally, but CI needs to pass with flake8. Flake8 includes a bunch of other ones as you found out, and the configuration currently doesn't break anything, but removing it may break some of people's environments. This is all hypothetical, which is why I'm asking to check wit your peers ;)